From 9f8db28e255f4bd429dcc6a9ed018e9283d02f08 Mon Sep 17 00:00:00 2001
From: Constantin Musca <constantin.musca@intel.com>
Date: Wed, 11 Feb 2015 17:51:31 +0200
Subject: [PATCH] inv_mpu6050: set INV_MPU6050_TIME_STAMP_TOR to 1

At the beginning of a data stream of events there is usually a
timestamp mismatch at the first or second position.
Example:
timestamp: Delta (t[n] - t[n-1])
978729834133970000  233033344
978729834367003000  5015680
978729834372019000  4928384
The timestamps kfifo can hold timestamps from old interrupts
which will be reported for new interrupts.
So, reduce INV_MPU6050_TIME_STAMP_TOR to 1 to flush the
fifo earlier.

Change-Id: I300568431aff597046a29ecc06da9acdc3f811bb
Tracked-On: https://jira01.devtools.intel.com/browse/IRDA-3689
Signed-off-by: Constantin Musca <constantin.musca@intel.com>
---
 drivers/iio/imu/inv_mpu6050/inv_mpu_iio.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/iio/imu/inv_mpu6050/inv_mpu_iio.h b/drivers/iio/imu/inv_mpu6050/inv_mpu_iio.h
index deb544c..a461423 100644
--- a/drivers/iio/imu/inv_mpu6050/inv_mpu_iio.h
+++ b/drivers/iio/imu/inv_mpu6050/inv_mpu_iio.h
@@ -194,7 +194,7 @@ struct inv_mpu6050_state {
 
 /* init parameters */
 #define INV_MPU6050_INIT_FIFO_RATE           50
-#define INV_MPU6050_TIME_STAMP_TOR                        5
+#define INV_MPU6050_TIME_STAMP_TOR                        1
 #define INV_MPU6050_MAX_FIFO_RATE                         1000
 #define INV_MPU6050_MIN_FIFO_RATE                         4
 #define INV_MPU6050_ONE_K_HZ                              1000
-- 
1.9.1

